service/storage_service.cc

```diff
@@ -2722,7 +2723,7 @@ future<> storage_service::restore_replica_count(inet_address endpoint, inet_addr
         }
         streamer->add_rx_ranges(keyspace_name, std::move(ranges_per_endpoint));
     }
-    return streamer->stream_async().then_wrapped([this, streamer, notify_endpoint] (auto&& f) {
+    streamer->stream_async().then_wrapped([this, streamer, notify_endpoint] (auto&& f) {
```

Comment:
Not critical, but if we run inside a thread then it's much nicer to to have a mix of threads and continuation:

```c++
    try {
        streamer->stream_async().get();
    } catch (...) {
        slogger.warn(...);
    }
    send_replication_notification.get();
```